easybuild/easyconfigs/a/ASE/ASE-3.22.1-foss-2022b.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.22.1'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```

Comment:
Any reason why this is not at the `gfbf` level? It seems like it can be...?

Reply:
When I wrote it, matplotlib was still a PR and was foss/2022b, and I did not notice that it had changed before I submitted this one.  Fixing it!  Thanks, @casparvl

Reply:
Great, thanks!